lib/solr/commit/request.rb

```diff
@@ -3,9 +3,16 @@ module Commit
     class Request
       PATH = '/update'.freeze
 
-      def run
-        http_request = Solr::Request::HttpRequest.new(path: PATH, url_params: { commit: true }, method: :post)
-        Solr::Request::Runner.call(request: http_request)
+      def run(runner_options: nil)
+        http_request = Solr::Request::HttpRequest.new(path: PATH, url_params: { commit: true, openSearcher: true }, method: :post)
```

Comment:
`openSearcher` default value is `true`

Reply:
Yep there's another part I'm pushing now, it makes use of this option